### DISCUSSION TITLE
Can't export promise from `eslintrc.js`?

### DISCUSSION THREAD

**Discussion by cha0s** [eslint/eslint#15721] [Q&A]
It appears that ESLint can't be configured asynchronously. Am I correct, or am I screwing up something else on my end?

In other words, I want:

`.eslintrc.js`:
```javascript
module.exports = new Promise(...);
```

**Comment by nzakas** [comment]
You’re correct. You must return an object. Promises don’t work.

**Comment by cha0s** [comment]
Are you accepting pull requests for such a feature?

**Reply by nzakas** [reply]
No. There is no way to make such a thing work currently.

**Reply by cha0s** [reply]
You seem very sure about that.

Very well, I won't waste my effort.
